styles.css

```
@@ -0,0 +1,39 @@
+.caixa {
+    display:flex;
+    flex-direction: column;
+    margin-bottom: 10px;
+    padding: 10px;
+}
+
+h1 {
+    text-align: center;
+    font-family: Tahoma;
+    font-weight: 700;
+    color: #ffefb9;
+}
+
+body {
+    background-color: #44355b;
+    font-family: tahoma;
+}
+
+p{
+color: #eca72c;
+text-align: center;
+}
```

```suggestion
p{
  color: #eca72c;
  text-align: center;
}
```

É sempre bom mantermos coerencia na identação do nosso código, tanto para facilitar a leitura quanto para manter o arquivo padronizado